### PR TITLE
Temporarily disabled unnormalized-clamp-linear-float.cpp

### PR DESCRIPTION
**Pull request by DenisBakhvalov**
Related issue: intel/llvm#8814

### CODE REVIEW

**Comment by cperkinsintel** [comment]
@DenisBakhvalov  - this test is already marked XFAIL for LevelZero and Windows.  Why is it necessary to further disable it?   If we don't want the test run at all, we can switch `XFAIL:` to `UNSUPPORTED:`

**Comment by DenisBakhvalov** [comment]
@cperkinsintel, based on what I saw, it sporadically passes on Windows + LevelZero, even though it is supposed to fail.

**Comment by cperkinsintel** [comment]
@DenisBakhvalov , well let's just switch `XFAIL:` to `UNSUPPORTED:` if "sometimes passing" is a problem. Then the test won't be run at all. 
Adding an outside requirement just makes it more complicated, especially since we want to maintain this test for other devices and platforms to catch any regressions.  There are outstanding tickets for this problem. Once the L0 team get their fixes in, it'd be nice to be able to reactive this test easily.

**Review by bader — DISMISSED** [review]

File: SYCL/Sampler/unnormalized-clamp-linear-float.cpp [hunk 1]
```diff
@@ -10,6 +10,9 @@
 // On Windows, LevelZero returns wrong result for outside right pixel when using
 // unorm_int8 data.
 
+// #204 Disable unstable tests
+// REQUIRES: TEMPORARY_DISABLED
```
Comment (bader) [inline SYCL/Sampler/unnormalized-clamp-linear-float.cpp]: Probably it should be disabled only for one configuration: Windows + Level Zero.
I think we still want to test all other configuration. Right?

**Comment by DenisBakhvalov** [comment]
> @DenisBakhvalov , well let's just switch `XFAIL:` to `UNSUPPORTED:` if "sometimes passing" is a problem. Then the test won't be run at all.
> Adding an outside requirement just makes it more complicated, especially since we want to maintain this test for other devices and platforms to catch any regressions. There are outstanding tickets for this problem. Once the L0 team get their fixes in, it'd be nice to be able to reactive this test easily.

OK.

File: SYCL/Sampler/unnormalized-clamp-linear-float.cpp (hunk 1 above)
Comment (DenisBakhvalov) [reply SYCL/Sampler/unnormalized-clamp-linear-float.cpp]: Changed it as Chris suggested.

File: SYCL/Sampler/unnormalized-clamp-linear-float.cpp [hunk 2]
```diff
@@ -2,7 +2,8 @@
 // RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
 // RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
 // RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
-// XFAIL: cuda || (level_zero && windows)
+// Temporary disabled on level_zero && windows (#204)
+// UNSUPPORTED: cuda || (level_zero && windows)
```
Comment (bader) [inline SYCL/Sampler/unnormalized-clamp-linear-float.cpp]: This change disables execution of the test on CUDA. 
Current version runs the test, but expects non-zero return code. New version doesn't run the test on CUDA at all.

Comment (DenisBakhvalov) [reply SYCL/Sampler/unnormalized-clamp-linear-float.cpp]: Right. Fixed.

**Review by bader — APPROVED** [review]

**Review by cperkinsintel — APPROVED** [review]
LGTM.  Thanks!

**Comment by dm-vodopyanov** [comment]
Just ran checks in intel/llvm-test-suite#202 one more time, and now this test fails with OpenCL on Linux:
http://134.191.240.166:8010/blue/organizations/jenkins/LLVM-Test-Suite-CI%2FLLVM-Test-Suite-Lin/detail/LLVM-Test-Suite-Lin/455/pipeline#step-177-log-475

**Comment by DenisBakhvalov** [comment]
> Just ran checks in intel/llvm-test-suite#202 one more time, and now this test fails with OpenCL on Linux:
> http://134.191.240.166:8010/blue/organizations/jenkins/LLVM-Test-Suite-CI%2FLLVM-Test-Suite-Lin/detail/LLVM-Test-Suite-Lin/455/pipeline#step-177-log-475

I think it's a different one. The failing test in your CI is `Sampler/normalized-none-linear-float.cpp`. Nevertheless, the issue seems a little annoying.

**Comment by cperkinsintel** [comment]
@dm-vodopyanov  - I'll take care of it. We had to adjust the nonnormalized test last week. I'll update this one too. The OCL:CPU has an error where it gets flaky results on certain edge cases. There's an outstanding ticket on it, but I'll adjust the test in the interim.

**Comment by dm-vodopyanov** [comment]
Thanks a lot Chris! Denis, yes, sorry, this is completely another test, I didn't match the whole names.

**Review by v-klochkov — APPROVED** [review]
